hugo/layouts/partials/footer.html

```diff
@@ -0,0 +1,10 @@
+{{ if default true $.Params.include_footer_slack }}
```

Comment:
I'd convert to opt-in the `slack` and `email` configurations because that way it is more explicit the components that will be shown in the footer part for each page.
What do you think @Serabe ?